Makefile

```diff
@@ -415,6 +424,11 @@ ci-job-compilation:
 	$(call banner,CI-Job: Compilation)
 	@NOWARNINGS=true $(MAKE) allboards
 
+.PHONY: ci-job-msrv
+ci-job-msrv: ci-job-cargo-hack
+	$(call banner,CI-Job: MSRV Check)
+	@cd boards/hail && cargo hack check --rust-version --target thumbv7em-none-eabihf
```

Comment:
This automatically compares against the version in hail's `Cargo.toml` I suppose?